src/popup/__mocks__/PopupLayer.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export default class PopupLayer extends React.Component<{ children: React.ReactNode }> {
+    name = 'PopupLayerMock';
```

Comment:
Is this variable being used?

Reply:
Yeah I think it's used when it's rendered